tests/core/test_reader.py

```diff
@@ -162,3 +249,25 @@ def sys_exit(*_args):
             raise Exception("should not get here")
 
     assert "exit" in str(e.value)
+
+
+def test_message_reader(tmp_path):
```

Comment:
you are not using the `tmp_path` fixture on this test, so there is no need to call it

Reply:
Good catch - removed in the "add basic test" commit.